packages/browser-destinations/destinations/google-analytics-4-web/src/__tests__/setConfigurationFields.test.ts

```diff
@@ -615,4 +637,52 @@ describe('Set Configuration Fields action', () => {
     setConfigurationEvent.page?.(context)
     expect(mockGtag).toHaveBeenCalledWith('consent', 'update', {})
   })
+  it('should update config if payload has send_page_view undefined', () => {
+    const context = new Context({
+      event: 'setConfigurationFields',
+      type: 'page',
+      properties: {
+        send_page_view: undefined
+      }
+    })
+
+    setConfigurationEvent.page?.(context)
+    expect(mockGtag).toHaveBeenCalledWith('config', 'G-XXXXXXXXXX', {
+      allow_ad_personalization_signals: false,
+      allow_google_signals: false,
+      send_page_view: true
+    })
+  })
+  it('should update config if payload has send_page_view true', () => {
```

Comment:
Shouldn't this scenario not update config? I assume send_page_view is true by default.
```suggestion
  it('should not update config if payload has send_page_view true', () => {
```